tests/CommonFunctionTest.php

```diff
@@ -0,0 +1,112 @@
+<?php declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+
+final class CommonFunctionTest extends TestCase
+{
+    /**
+     * @covers t
+     */
+    public function testT(): void
+    {
+        $actual = t("foo\tbar\tbaz");
+
+        $this->assertEquals('foobarbaz', $actual);
+    }
```

Comment:
Test `t()` function and check if the return value of the `t()` function is as expected.